modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/SpringCodegen.java

```diff
@@ -590,6 +598,16 @@ public void processOpts() {
         }
     }
 
+    private boolean containsEnums() {
```

Comment:
Proposal for more better readability. WDYT?

```
   private boolean containsEnum() {
      if (openAPI == null) {
         return false;
      }
      
      Components components = openAPI.getComponents();
      if (components == null || components.getSchemas() == null) {
         return false;
      }
      
       return components.anyMatch(component -> component.getEnum() != null && !component.getEnum().isEmpty());
      }
```

Reply:
Also, further thinking about this, would be good to add unit tests for this method

Reply:
I've added a test, but it's more a test-by-side-effect with the file creation.